variants/rak2560/platformio.ini

```diff
@@ -0,0 +1,24 @@
+; The very slick RAK wireless RAK 4631 / 4630 board - Unified firmware for 5005/19003, with or without OLED RAK 1921
+[env:rak2560]
+extends = nrf52840_base
+board = wiscore_rak4631
+board_check = true
+build_flags = ${nrf52840_base.build_flags} -Ivariants/rak2560 -D RAK_4631
+  -L "${platformio.libdeps_dir}/${this.__env__}/bsec2/src/cortex-m4/fpv4-sp-d16-hard"
+  -DGPS_POWER_TOGGLE ; comment this line to disable triple press function on the user button to turn off gps entirely.
+  -DHAS_RAKPROT=1 ; Define if RAk OneWireSerial is used (disables GPS)
+build_src_filter = ${nrf52_base.build_src_filter} +<../variants/rak2560> +<mesh/eth/> +<mesh/api/> +<mqtt/>
```

Comment:
src/platform/nrf52/architecture.h needs to be amended to pick up the new hardware model, before the plain 4630 is found.